### DISCUSSION TITLE
Unordered Groups

### DISCUSSION THREAD

**Discussion by marcfir** [pest-parser/pest#859] [Q&A]
Is it possible to describe unordered groups like in [xtext](https://www.eclipse.org/Xtext/documentation/301_grammarlanguage.html#unordered-groups)?

E.g. `UG( 'static' | 'final' | 'private')`.

**Comment by tomtau** [comment]
Is this related: https://github.com/pest-parser/pest/discussions/842 ?

**Reply by marcfir** [reply]
Yes, this is the same question.
